networking/manager.go

```diff
@@ -817,6 +828,40 @@ func (this *networkManagerInstance) submitConfig(config *maestroSpecs.NetworkCon
 		}
 
 	}
+
+	//id of the dns in db
+	dnsid := string("nameservers")
+
+	//the stored db
+	var storedconfig []string
+
+	//get the stored dns servers
+	err := this.networkConfigDB.Get(dnsid, &storedconfig)
+
+	if config.Existing == "replace" {
+		//erase and replace the nameservers
+		storedconfig = nil
+		storedconfig = append(storedconfig, config.Nameservers...)
+	} else if config.Existing == "override" || config.Existing == "" {
+		//merge the new into the ould
+		for i := range config.Nameservers {
+			if indexOf(storedconfig, config.Nameservers[i]) == -1 {
+				storedconfig = append(storedconfig, config.Nameservers[i])
+			}
+		}
+	}
+
+	//store name servers
+	err = this.networkConfigDB.Put(dnsid, storedconfig)
+	if err != nil {
+		log.MaestroErrorf("NetworkManager: unable to store nameservers")
+	} else {
+		//make active
```

Comment:
this already occurs in applyDNS(), which means that it will end up being called twice.  do we need it in both places?